docs/api-reference.md

```diff
@@ -405,18 +405,432 @@ await sideEffects.scheduleCommand(
 
 <!-- prettier-ignore-end -->
 
+## Event Store Adapter API
+
+An event store adapter defines how the reSolve framework stores events in the underlying event store. An event store adapter object must expose the following functions:
```

Comment:
Is it "reSolve" or "ReSolve"? Please make sure that capitalization is consistent throughout the document.

Reply:
It is "reSolve". I fixed all occurences of wrong capitalization.

Reply:
Thank you.